JavaAssignment/src/rashmi/StudentMarks.java

```diff
@@ -0,0 +1,29 @@
+package rashmi;
+
+import java.util.Scanner;
+
+public class StudentMarks 
+{
+	void studentGrade(int marks)
+	{
+		if(marks>=75 && marks<=100)
+		{
+			System.out.println("Distinction");
+		}
+		else if(marks>=50 && marks<75)
+		{
+			System.out.println("First Class");
+		}
+		else if(marks<50)
+		{
+			System.out.println("Fail");
+		}
+	}
```

Comment:
Here, we need add one more "else" for invalid input like -ve marks & more than 100 marks.